packages/opencensus-web-all/src/export-initial-load.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {adjustPerfTimeOrigin, tracing} from '@opencensus/web-core';
+import {OCAgentExporter} from '@opencensus/web-exporter-ocagent';
+import {clearPerfEntries, getInitialLoadRootSpan, getPerfEntries} from '@opencensus/web-instrumentation-perf';
+
+import {OpenCensusWebConfig, WindowWithOcwGlobals} from './types';
+
+const windowWithOcwGlobals = window as WindowWithOcwGlobals;
+
+/** How long to wait after `load` event to export initial load spans. */
+const WAIT_TIME_AFTER_LOAD_MS = 2000;
```

Comment:
Just curious to know about this value.

Reply:
I added a comment to explain that this delay is to allow other after-load handlers to fire first so that this does not slow down the user experience.